DQM/SiPixelPhase1Heterogeneous/python/SiPixelPhase1HeterogenousDQM_FirstStep_cff.py

```diff
@@ -1,10 +1,30 @@
 import FWCore.ParameterSet.Config as cms
-from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorTrackSoA_cfi import *
-from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorVertexSoA_cfi import *
-from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorRecHitsSoA_cfi import *
+from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorTrackSoA_cfi import siPixelPhase1MonitorTrackSoA as _siPixelPhase1MonitorTrackSoA
+from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorVertexSoA_cfi import siPixelPhase1MonitorVertexSoA as _siPixelPhase1MonitorVertexSoA
+from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorRecHitsSoA_cfi import siPixelPhase1MonitorRecHitsSoA as _siPixelPhase1MonitorRecHitsSoA
 
-from Configuration.ProcessModifiers.gpu_cff import gpu
-gpu.toModify(siPixelPhase1MonitorRecHitsSoA, pixelHitsSrc = "siPixelRecHitsPreSplittingSoA")
+siPixelPhase1MonitorRecHitsSoA = _siPixelPhase1MonitorRecHitsSoA.clone()
+siPixelPhase1MonitorVertexSoA = _siPixelPhase1MonitorVertexSoA.clone()
+siPixelPhase1MonitorTrackSoA = _siPixelPhase1MonitorTrackSoA.clone()
 
+from Configuration.ProcessModifiers.gpu_cff import gpu
+gpu.toModify(siPixelPhase1MonitorRecHitsSoA, pixelHitsSrc = "siPixelRecHitsPreSplittingSoA") #would be obsloete if .501 is dropped
 
 monitorpixelSoASource = cms.Sequence(siPixelPhase1MonitorRecHitsSoA * siPixelPhase1MonitorTrackSoA * siPixelPhase1MonitorVertexSoA)
+
+## GPU vs CPU validation
+from Configuration.ProcessModifiers.gpuValidationPixel_cff import gpuValidationPixel
```

Comment:
@AdrianoDee sorry, but where is this created? 
Btw, it'll conflict with  https://github.com/cms-sw/cmssw/pull/37619 ?